docs/api/cookies.md

```diff
@@ -96,6 +96,7 @@ the response.
   * `expirationDate` Double (optional) - The expiration date of the cookie as the number of
     seconds since the UNIX epoch. If omitted then the cookie becomes a session
     cookie and will not be retained between sessions.
+  * `sameSite` String (optional) - The [Same Site](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#SameSite_cookies) policy to apply to this cookie.  Can be `unspecified`, `norestriction`, `lax` or `strict`.  Default is `norestriction`.
```

Comment:
`no_restriction` (with an underscore) is the string used by the WebExtension API, should this API match that since it's already a convention? https://developer.chrome.com/extensions/cookies#type-SameSiteStatus

Reply:
@flotwig Thanks for the link, that's actually a good reason to put it back to having an underscore 👍